source/post.mysimulation.cfg

```diff
@@ -1,58 +1,114 @@
 [default]
-input = /lcrc/group/e3sm/ac.golaz/E3SM_simulations/20210305.v2beta3.piControl.ne30pg2_EC30to60E2r2.chrysalis
-output = /lcrc/group/e3sm/ac.golaz/E3SM_simulations/20210305.v2beta3.piControl.ne30pg2_EC30to60E2r2.chrysalis
-case = 20210305.v2beta3.piControl.ne30pg2_EC30to60E2r2.chrysalis
-www = /lcrc/group/e3sm/public_html/diagnostic_output/ac.golaz/E3SM/v2/beta
+# The directory with the data to use
+input = /lcrc/group/e3sm/ac.forsyth2/E3SM_simulations/20210122.v2_test01.piControl.ne30pg2_EC30to60E2r2-1900_ICG.chrysalis
+# The subdirectory with the eam files
+input_subdir = archive/atm/hist
+# Where the post-processing results should go
+output = /lcrc/group/e3sm/ac.forsyth2/E3SM_simulations/20210122.v2_test01.piControl.ne30pg2_EC30to60E2r2-1900_ICG.chrysalis
```

Comment:
Should we just set `output` and `www` to `<output>` and `<www>` respectively, as to not have users inadvertently overwrite my files?